public class FunctionUse5745 {

    public void funcUse() {
		org.apache.hadoop.io.SecureIOUtils.AlreadyExistsException alreadyexistsexception = new org.apache.hadoop.io.SecureIOUtils.AlreadyExistsException();
    }
}